java/src/main/java/com/google/appengine/tools/pipeline/impl/backend/AppEngineTaskQueue.java

```diff
@@ -120,20 +119,23 @@ public TaskReference enqueue(PipelineTask pipelineTask) {
   }
 
   @Override
-  public TaskReference enqueue(String queueName, TaskSpec build) {
-    log.finest("Enqueueing: " + build);
-    TaskOptions taskOptions = toTaskOptions(build);
+  public Collection<TaskReference> enqueue(String queueName, Collection<TaskSpec> taskSpecs) {
```

Comment:
implemented a collection case in the interface and leave the single case as a particular usage of that one.